tests/utils/exec/idps.go

```diff
@@ -9,12 +9,15 @@ import (
 )
 
 type IDPArgs struct {
-	ClusterID     string        `json:"cluster_id,omitempty"`
-	Name          string        `json:"name,omitempty"`
-	ID            string        `json:"id,omitempty"`
-	Token         string        `json:"token,omitempty"`
-	OCMENV        string        `json:"ocm_environment,omitempty"`
-	URL           string        `json:"url,omitempty"`
+	ClusterID string `json:"cluster_id,omitempty"`
+	Name      string `json:"name,omitempty"`
+	ID        string `json:"id,omitempty"`
+	Token     string `json:"token,omitempty"`
```

Comment:
Run "terraform fmt" , please

Reply:
It's go code. You can install extension to keep code fmt alignment automatically.

Reply:
which extension? for VS Code?

Reply:
I have `Go Grammar` and "Go Outliner" installed.